client/src/views/Simulation/components/ParametersPanel.vue

```diff
@@ -59,7 +59,7 @@
           class="parameter"
           v-for="(parameter, index) of displayedParameters"
           :key="index"
-          :class="{ hidden: parameter.hidden }"
+          :class="[{ hidden: parameter.hidden, highlighted: parameter.metadata.name === highlighted }]"
```

Comment:
You do not need to use an array, the object is enough:
```
:class="{
  hidden: parameter.hidden,
  highlighted: parameter.metadata.name === highlighted,
}"
```

Reply:
Thank you. Done.